### ISSUE TITLE
Windows 8 x64 crash

### ISSUE THREAD

**Issue by shirajdesilva** [sdarnell/meteor-win#6]
We try to run a project on Windows 8 (64 bit) with 0.6.3.1 - rev 3 but it will crash as soon as it starts (without an error)

```
C:\Users\Danielle\Documents\GitHub\ak500 [master +0 ~1 -0]> dir


    Directory: C:\Users\Danielle\Documents\GitHub\ak500


Mode                LastWriteTime     Length Name
----                -------------     ------ ----
d----        25/05/2013  11:11 PM            .meteor
d----        25/05/2013  11:08 PM            client
d----        25/05/2013  11:08 PM            models
d----        25/05/2013  11:08 PM            packages
d----        25/05/2013  11:08 PM            public
d----        25/05/2013  11:08 PM            server
-a---        25/05/2013   2:58 PM       2889 .gitignore
-a---        23/05/2013   9:20 PM        263 .project
-a---        23/05/2013   7:19 PM        834 README.md


C:\Users\Danielle\Documents\GitHub\ak500 [master +0 ~1 -0]> meteor
=> Running Meteor from a checkout -- overrides project version (0.6.3.1)

[[[[[ ~\Documents\GitHub\ak500 ]]]]]

=> Meteor server running on: http://localhost:3000/
C:\Users\Danielle\Documents\GitHub\ak500 [master +0 ~1 -0]>
```

It stops after running (I didn't stop it)

**Comment by sdarnell** [comment]
I'll try and get hold of a windows 8 x64 environment, but it'll be about a week as I'm out of town.
In the mean time, does the same thing happen if you create and run one of the examples? E.g. todos
Could you also check the window event log for application crash events.
Another thing to try is to uninstall the rev3 build and try the first 0.6.3.1 build (before the multi-mongo changes).

**Comment by shirajdesilva** [comment]
Resolved. Needed to change node.exe in Meteor bin folder to Windows 7 compatibility. Works fine then. Thanks.

**Comment by sdarnell** [comment]
I also notice you have a packages directory. Are all of the packages pure JavaScript? (i.e. no native extensions).

**Comment by sdarnell** [comment]
For your information I have just released 0.6.3.1-rev4 which automatically creates the registry key that sets Win7 compatibility mode for node.
